### DISCUSSION TITLE
Horizontal stack problem

### DISCUSSION THREAD

**Discussion by edugarciavero** [matt8707/ha-fusion#382] [General]
I don't know if I'm doing something wrong, but I can't add anything inside the horizontal bars. Some error?
![hz stack](https://github.com/matt8707/ha-fusion/assets/129856554/d2244807-cd83-4b92-bdeb-eccd1b1d0625)

**Comment by matt8707** [comment]
drag a section to it

**Reply by edugarciavero** [reply]
I tried every way

https://github.com/matt8707/ha-fusion/assets/129856554/18ff71be-4e35-45e1-946d-bba894f52a72

**Reply by matt8707** [reply]
drag "Luzes" into hstack, not individual objects

**Reply by edugarciavero** [reply]
Perfect! Thank you very much. Last question, is it possible to move a button to the side bar?
